core/blocks/class-image-maxi-block.php

```
@@ -8,627 +8,770 @@
+				),
+				'position' => get_position_styles(
+					get_group_attributes($props, 'position'),
+				),
+				'display' => get_display_styles(
+					get_group_attributes($props, 'display'),
+				),
+				'alignment' => get_alignment_flex_styles(
+					get_group_attributes($props, 'alignment'),
+				),
+				'size' => get_size_styles(get_group_attributes($props, 'size')),
+				$fit_parent_size,
```

PHPMD (2.15.0)

Avoid unused local variables such as '$fit_parent_size'. (undefined)

(UnusedLocalVariable)